packages/core/lib/command-utils.js

```diff
@@ -201,6 +203,41 @@ const runCommand = async function (command, options) {
   return await command.run(options);
 };
 
+const processHelpInput = inputStrings => {
+  //is it wrong syntax of help?
```

Comment:
what does this comment mean exactly?

Reply:
I will change this comment.  Would this work `When incorrect input syntax is used to access help information`?